content/posts/manage-list-feature-flags-with-file/index.md

```diff
@@ -0,0 +1,229 @@
++++
+date = "2021-12-29T14:28:23+07:00"
+author = "nhattien.ho"
+description = "Manage list feature flags with file yaml/json."
+title = "Create feature flags using file in Unleash"
+categories = ["Feature toggle", "Testing"]
+tags = ["Feature toggle", "Unleash", "Node.js"]
+slug = "manage-list-flags-with-file"
++++
+
+In this blog, we will learn how to create a list of feature toggles in Unleash using file. First, let's learn some concept of feature toggle and Unleash 
```

Comment:
Have you ever struggle having an overview of the feature toggles across your services and applications, especially when you have a lot on the table? We've been there before!
In this blog, let's learn how to create a list of feature toggles in Unleash using file. 
But first, let's learn some concept of feature toggle and Unleash